opengrok-web/src/main/webapp/js/searchable-option-list-2.0.13.js

```diff
@@ -1093,9 +1097,14 @@
                         /*
                          * Modified for OpenGrok in 2017.
                          */
-                        if (isOnSearchPage()) {
-                            $('#xrd').val("1"); // no redirect
-                            $('#sbox').submit();
+                        if (self.config.quickDeleteForm) {
+                            if (self.config.quickDeletePermit) {
```

Comment:
I saw the original author using

`$.isFunction(this.config.events.onChange)`

is it worth to keep it consistent?

Reply:
I'm a JS noob. Can you clarify?

Reply:
I mean that lower below this code the original author wrote a check for function existence like

```
                if ($.isFunction(this.config.events.onChange)) {
                    this.config.events.onChange.call(this, this, $changedInputs);
                }
```

so perhaps we want to use it as well like

```
if ($.isFunction(self.config.quickDeletePermit))
```

to be consistent with the author's style

Reply:
however, looking at the doc https://api.jquery.com/jQuery.isFunction/ the function is deprecated, so keep it as you have it

Reply:
OK thank you